lisp/ess-custom.el

```diff
@@ -1282,11 +1287,12 @@ global) values from ESS indentation variables."
                  (const RStudio)
                  (const RStudio-)
                  (const DEFAULT))
-  :group 'ess-edit)
+  :group 'ess-edit
+  :safe #'ess-style-safe)
 
-;; the real setting of this happens via <foo>-editing-alist:
-(defvar ess-style ess-default-style
-  "Current ESS indentation style, see `ess-style-alist' for more.")
+(defun ess-style-safe (v)
```

Comment:
Internal function?

Reply:
I think it makes sense for this not to be internal. If people implement their own styles, they'll want to make sure they're treated as safe. WDYT?

Reply:
I think this function is useless to users. Safe is `ess-style` variable not individual styles. In order to be safe, aka assignable to `ess-style` a style need to be part of `ess-style-alist`.